### PR TITLE
Implement i64.div and i64.rem intrinsics

### PR DESCRIPTION
**Pull request by no1wudi**

### CODE REVIEW

File: core/iwasm/aot/aot_intrinsic.c [hunk 1]
```diff
@@ -514,6 +542,15 @@ add_intrinsic_capability(AOTCompContext *comp_ctx, uint64 flag)
     }
 }
 
+static void
+add_i64_common_intrinsics(AOTCompContext *comp_ctx)
```
Comment (lum1n0us) [inline core/iwasm/aot/aot_intrinsic.c:546]: who is going to call this function?

**Review by lum1n0us — COMMENTED** [review]
Just one question

File: core/iwasm/aot/aot_intrinsic.c (hunk 1 above)
Comment (no1wudi) [reply core/iwasm/aot/aot_intrinsic.c:546]: I guess all 32-bit platform will need it, but only riscv32 verified now.

**Comment by lum1n0us** [comment]
LGTM

**Pull request merged**